style.css

```diff
@@ -320,3 +338,252 @@ nav {
   margin-top: 50px;
   margin-bottom: 10px;
 }
+
+.snap01 {
+  display: none;
+}
+
+.list-1 {
+  list-style: none;
+}
+
+.list-item-1 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-2 {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100px;
+  list-style: none;
+}
+
+.list-item-2 {
+  background: #f7f7f9;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  margin-top: 20px;
+  border-radius: 5px;
+}
+
+.name {
+  margin-left: 10px;
+}
+
+.list-3 {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+.list-4 {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+@media screen and (min-width: 1024px) {
+  .logo {
+    margin-left: 60px;
+    margin-top: 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: inherit;
+    margin-right: 30px;
+  }
+
+  .nav-link {
+    text-decoration: none;
+    margin-right: 30px;
+    color: #344563;
+  }
+
+  .homepage {
+    height: 768px;
+    background-image: url(./images/background_1024.png);
+  }
+
+  .home-page {
+    width: 870px;
+    margin: 100px auto;
+  }
+
+  .intro {
+    width: 350px;
+    height: 122px;
+  }
+
+  /* .description {
+    width: 870px;
+    height: 60px;
+  } */
+
+  .cr-grid {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .w1contain {
+    flex-direction: row;
+    justify-content: space-between;
+    align-self: center;
+  }
```

Comment:
- [ ] Consider adding `cursor: pointer` to all your project buttons on hover to improve user experience and interaction.

- [ ] Currently, your image for the first project is broken. Consider working on it to restore it and try to match your design template so that your work looks professional.


Your design:
<img width="761" alt="chrome_ic04lnCOhr" src="https://user-images.githubusercontent.com/81468675/144472424-7a63a941-5e44-4e9b-b446-b40c30cc204d.png">

Figma design:
<img width="259" alt="chrome_sYXRHMI9Yj" src="https://user-images.githubusercontent.com/81468675/144472463-ea4f425a-b071-4bc1-b0b3-dd50e9cddec9.png">

Reply:
Thanks for reaching out and telling me about my mistakes and flaws. I just want to say I checked my project on multiple computers of my friends and microverse friends and for them these images are visible. I picked these images straight from Figma and insert all images inside my code with images. There is no complex code for adding image and i don't know why images are not visible to you. Can you please check my project on another browser or maybe on another machine? 
Please......................................................
I'll be very thankful to you.

Reply:
Hi @HAMMAS-SALEEM

I just checked your work on the terminal now and it seems to be working fine. However, the site we use to check the HTML page is not displaying the image. But since the terminal is displaying the image then we are good to go. Sorry for any inconveniences this might have caused you 🥂